examples/image-streams/image-streams-centos7.json

```diff
@@ -22,6 +22,7 @@
               "description": "Build and run Ruby 2.0 applications",
               "iconClass": "icon-ruby",
               "tags": "builder,ruby",
+              "supports": "ruby:2.0,ruby",
```

Comment:
Why do you need supports "ruby:2.0"?  Shouldn't the lookup algorithm be to "ruby:2.0" automatically from the name of the tag and the name of the stream?  Or are you just being explicit?